src/Microsoft.Xaml.Behaviors/Microsoft.Xaml.Behaviors.csproj

```diff
@@ -1,197 +1,98 @@
-    <Compile Include="TypeConverterHelper.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="VisualStateUtilities.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="..\RuntimeVersion.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="..\Version\Version.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <EmbeddedResource Include="ExceptionStringTable.resx">
-      <Public>True</Public>
-      <Generator>ResXFileCodeGenerator</Generator>
-      <LastGenOutput>ExceptionStringTable.Designer.cs</LastGenOutput>
-    </EmbeddedResource>
+
+  <ItemGroup Condition="'$(TargetFramework)'=='net45'">
     <Reference Include="System" />
     <Reference Include="System.Core" />
     <Reference Include="System.Xaml" />
     <Reference Include="System.XML" />
     <Reference Include="WindowsBase">
       <Private>False</Private>
-      <HintPath>$(WindowsPath)\WindowsBase.dll</HintPath>
     </Reference>
     <Reference Include="PresentationCore">
       <Private>False</Private>
-      <HintPath>$(WindowsPath)\PresentationCore.dll</HintPath>
     </Reference>
     <Reference Include="PresentationFramework">
       <Private>False</Private>
-      <HintPath>$(WindowsPath)\PresentationFramework.dll</HintPath>
     </Reference>
-    <AppDesigner Include="obj\Properties\" />
   </ItemGroup>
-  <ItemGroup>
-    <None Include="FinalPublicKey.snk" />
-  </ItemGroup>
-  <ItemGroup>
+
+  <ItemGroup Condition="'$(SignType)' == 'real'">
     <FilesToSign Include="$(OutputPath)$(AssemblyName).dll">
       <Authenticode>Microsoft</Authenticode>
       <StrongName>67</StrongName>
     </FilesToSign>
   </ItemGroup>
+
   <ItemGroup>
-    <PackageReference Include="MicroBuild.Core">
+    <Compile Include="..\RuntimeVersion.cs" Link="RuntimeVersion.cs" />
+    <Compile Include="..\Version\Version.cs" Link="Version.cs" />
+
+    <Compile Update="ExceptionStringTable.Designer.cs">
+      <SubType>Code</SubType>
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+      <DependentUpon>ExceptionStringTable.resx</DependentUpon>
+    </Compile>
+    <EmbeddedResource Update="ExceptionStringTable.resx">
+      <Public>True</Public>
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>ExceptionStringTable.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta-63127-02" PrivateAssets="All" />
+      <PackageReference Include="MicroBuild.Core">
       <Version>0.3.0</Version>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="NuGet.Build.Tasks.Pack">
-      <Version>4.8.0</Version>
-    </PackageReference>
   </ItemGroup>
-  <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
+
   <Target Name="TimestampNugetPackage">
```

Comment:
I would use Nerdbank.GitVersioning for this, not a timestamp. Also, SemVer2 based format for proper sorting.

Reply:
I agree but figure that should be a separate issue (this was already there). I'm already rocking the boat a lot with this PR 😁